docs/Manual/Tutorials/Kubernetes/bare-metal.md

```diff
@@ -0,0 +1,457 @@
+[bootstrap-token] Using token: blcr1y.49wloegyaugice8a
+[bootstrap-token] Configuring bootstrap tokens, cluster-info ConfigMap, RBAC Roles
+[bootstraptoken] configured RBAC rules to allow Node Bootstrap tokens to post CSRs in order for nodes to get long term certificate credentials
+[bootstraptoken] configured RBAC rules to allow the csrapprover controller automatically approve CSRs from a Node Bootstrap Token
+[bootstraptoken] configured RBAC rules to allow certificate rotation for all node client certificates in the cluster
+[bootstraptoken] creating the "cluster-info" ConfigMap in the "kube-public" namespace
+[addons] Applied essential addon: CoreDNS
+[addons] Applied essential addon: kube-proxy
+
+Your Kubernetes master has initialized successfully!
+
+To start using your cluster, you need to run the following as a regular user:
+
+  mkdir -p $HOME/.kube
+  sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+  sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+You should now deploy a pod network to the cluster.
+Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
+  https://kubernetes.io/docs/concepts/cluster-administration/addons/
+
+You can now join any number of machines by running the following on each node
+as root:
+
+kubeadm join 192.168.10.61:6443 --token blcr1y.49wloegyaugice8a --discovery-token-ca-cert-hash sha256:0505933664d28054a62298c68dc91e9b2b5cf01ecfa2228f3c8fa2412b7a78c8
+```
+
+Go ahead and do as above instructed and see into getting kubectl to work on the master:
+
+```
+kube01 > mkdir -p $HOME/.kube
+kube01 > sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+kube01 > sudo chown $(id -u):$(id -g) $HOME/.kube/config
+```
+
+## Deploy a pod network
+
+For this guide, we go with **flannel**, as it is an easy way of setting up a layer 3 network, which uses the Kubernetes API and just works anywhere, where a network between the involved machines works:
+
+```
+kube01 > kubectl apply -f \ 
+   https://raw.githubusercontent.com/coreos/flannel/bc79dd1505b0c8681ece4de4c0d86c5cd2643275/Documentation/kube-flannel.yml
+
+  clusterrole.rbac.authorization.k8s.io/flannel created
+  clusterrolebinding.rbac.authorization.k8s.io/flannel created
+  serviceaccount/flannel created
+  configmap/kube-flannel-cfg created
+  daemonset.extensions/kube-flannel-ds-amd64 created
+  daemonset.extensions/kube-flannel-ds-arm64 created
+  daemonset.extensions/kube-flannel-ds-arm created
+  daemonset.extensions/kube-flannel-ds-ppc64le created
+  daemonset.extensions/kube-flannel-ds-s390x created
+```
+
+## Join remaining nodes
+
+Run the above join commands on the nodes `kube02` and `kube03`. Below is the output on `kube02` for the setup for this guide:
+
+```
+kube02:~ > sudo kubeadm join 192.168.10.61:6443 --token blcr1y.49wloegyaugice8a --discovery-token-ca-cert-hash sha256:0505933664d28054a62298c68dc91e9b2b5cf01ecfa2228f3c8fa2412b7a78c8
```

Comment:
Placeholder for IP address?

Reply:
wrote the ip addresses into the introduction.